apps/omg_watcher/test/block_getter/core_test.exs

```diff
@@ -26,70 +26,74 @@ defmodule OMG.Watcher.BlockGetter.CoreTest do
 
   @eth Crypto.zero_address()
 
+  def assert_check(result, status, value) do
+    assert {^status, new_state, ^value} = result
+    new_state
+  end
+
+  def assert_check(result, value) do
+    assert {new_state, ^value} = result
+    new_state
+  end
+
   defp handle_downloaded_block(state, block) do
     assert {:ok, new_state, []} = Core.handle_downloaded_block(state, {:ok, block})
     new_state
   end
 
   test "get numbers of blocks to download" do
     start_block_number = 0
-    interval = 1_000
-    synced_height = 1
-    state = Core.init(start_block_number, interval, synced_height, maximum_number_of_pending_blocks: 4)
-
-    {state_after_chunk, block_numbers} = Core.get_numbers_of_blocks_to_download(state, 20_000)
-    assert block_numbers == [1_000, 2_000, 3_000, 4_000]
 
-    state_after_proces_down =
-      state_after_chunk
-      |> handle_downloaded_block(%Block{number: 4_000})
-      |> handle_downloaded_block(%Block{number: 2_000})
-
-    assert {_, [5_000, 6_000]} = Core.get_numbers_of_blocks_to_download(state_after_proces_down, 20_000)
+    start_block_number
+    |> Core.init(_interval = 1_000, _synced_height = 1, maximum_number_of_pending_blocks: 4)
```

Comment:
I think this `_interval = 1_000` isn't standard syntax (is it?) and is quite surprising, pls revert. Also note that it is now allowed to start `|>` with a function call.

Multiple times in the PR